Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -500,7 +500,7 @@ VideoConfigDiag::VideoConfigDiag(wxWindow* parent, const std::string &title, con
 	SettingCheckBox* efbcopy_disable = CreateCheckBox(page_hacks, _("Disable"), wxGetTranslation(efb_copy_desc), vconfig.bEFBCopyEnable, true);
 	efbcopy_texture = CreateRadioButton(page_hacks, _("Texture"), wxGetTranslation(efb_copy_texture_desc), vconfig.bCopyEFBToTexture, false, wxRB_GROUP);
 	efbcopy_ram = CreateRadioButton(page_hacks, _("RAM"), wxGetTranslation(efb_copy_ram_desc), vconfig.bCopyEFBToTexture, true);
-	cache_efb_copies = CreateCheckBox(page_hacks, _("Enable Cache"), wxGetTranslation(cache_efb_copies_desc), vconfig.bEFBCopyCacheEnable);
+	cache_efb_copies = CreateCheckBox(page_hacks, _("Speedhack"), wxGetTranslation(cache_efb_copies_desc), vconfig.bEFBCopyCacheEnable);
```

Comment:
I'll keep that in mind, thanks. Before opening the PR for full review including code style issues, i'll do that.